staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go

```diff
@@ -816,3 +816,14 @@ type ImportProxy struct {
 	// +optional
 	TrustedCAProxy *string `json:"trustedCAProxy,omitempty"`
 }
+
+// InfraNodePlacement provides information about replicas and placement for CDI components
+type InfraNodePlacement struct {
+	// nodePlacement describes scheduling configuration for specific CDI components
+	//+optional
+	*sdkapi.NodePlacement `json:",omitempty"`
```

Comment:
I think this property should be "sdkapi.NodePlacement `json:",inline"`" to be compatible

Reply:
Why? It is compatible right now

Reply:
The json is the same regardless of pointer or not, and maybe it doesn't cause any compiler errors but but with previous object a user of golang library could call:

```golang
ns := cdi.Spec.Infra.NodeSelector
```

Now, have to do:

```golang
var ns map[string]string
if  cdi.Spec.Infra != nil{
  ns = cdi.Spec.Infra.NodeSelector
}
```

Seems like unnecessary work to me.  Why deal with pointers if we don't have to when the json is the same?

Reply:
And to your point about interface with sdkapi, we would never passed nil before so I don't think we should care about that now